### DISCUSSION TITLE
Workshop 2021-11-29 feedback and questions

### DISCUSSION THREAD

**Discussion by holyjak** [holyjak/fulcro-intro-wshop#5] [General]
Your suggestions, feedback, questions regardign this workshop instance

**Comment by santi** [comment]
Overall I think this was a cool workshop - I definitely got a lot of new knowledge regarding Fulcro, and it was not too hard to follow the exercises.

One idea for improvement is to have more "challenges" for the reader during Part 1 of the workshop, where the reader is required to make some small changes to the code to reach a certain goal. Now it is mostly point and click with a recommendation to explore on your own, but personally I prefer some small tasks that "force" me to apply what I have just read. 

Other than that, I did not find any instructions before the event started regarding setup and preparations before the workshop started, so I used the first 10 minutes of the workshop to set everything up. I might have missed an email, but I think it should be included in the text on the Meetup page for ease of access.

Overall, great job, Jakub! I think your workshop at re:Clojure 2021 will be great 😊

**Reply by holyjak** [reply]
Thanks a lot for the feedback! And sorry about the Meetup page! I have sent two emails throug Meetups but perhaps they ended up in the spam folder ☹️ 
I will think about the excellent exercises feedback. Though I hope that the DIY coding exercises will most than satisfy your need at this!

**Comment by msolli** [comment]
Thanks for the workshop, Jakub! I appreciate all the work that must have gone into creating the material. It's really well thought out and high quality. I did not experience any issues following the instructions.

A couple of data points:
- I disobeyed your plea for using Calva - I used Cursive just fine. You mentioned the port numbers somewhere in the material, which made it easy for me to configure a "Remote REPL" to Shadow-CLJS and starting the CLJS REPL with `(shadow/repl :todomvc)`. If I hadn't made it work I would have use Calva as per instructions, I guess, but Cursive worked just fine.
- I did get your email from Meetup today before the workshop (at about 10:23 Oslo time).
